.github/workflows/bump-version-on-merge-next.yml

```diff
@@ -32,12 +32,11 @@ jobs:
         uses: codex-team/action-nodejs-package-info@v1
 
       # Stop workflow and do not bump version if it was changed already
-      - name: Stop workflow and do not bump version if it was changed already
-        uses: actions/github-script@v3
+      - name: Stop workflow if version was not changed
```

Comment:
Comment and name say opposite. What's the correct?

Reply:
Changed the action name, check now